Source/SpeechToTextV1/Tests/SpeechToTextTests.swift

```diff
@@ -277,6 +277,57 @@ class SpeechToTextTests: XCTestCase {
         }
         waitForExpectations()
     }
+    
+    // MARK: - Transcribe Data with Smart Formatting
+    
+    func testTranscribeStockAnnouncementCustomWAV() {
+        transcribeDataCustomForNumbers("StockAnnouncement", withExtension: "wav", format: .WAV)
+    }
+    
+    func transcribeDataCustomForNumbers(
+        filename: String,
+        withExtension: String,
+        format: AudioMediaType)
+    {
+        let description = "Transcribe an audio file with smart formatting."
+        let expectation = expectationWithDescription(description)
+        
+        let bundle = NSBundle(forClass: self.dynamicType)
+        guard let file = bundle.URLForResource(filename, withExtension: withExtension) else {
+            XCTFail("Unable to locate \(filename).\(withExtension).")
+            return
+        }
+        
+        guard let audio = NSData(contentsOfURL: file) else {
+            XCTFail("Unable to read \(filename).\(withExtension).")
+            return
+        }
+        
+        var settings = RecognitionSettings(contentType: format)
+        settings.continuous = true
+        settings.inactivityTimeout = 1
+        settings.keywords = ["stock"]
+        settings.keywordsThreshold = 0.75
+        settings.maxAlternatives = 3
+        settings.interimResults = true
+        settings.wordAlternativesThreshold = 0.25
+        settings.wordConfidence = true
+        settings.timestamps = true
+        settings.filterProfanity = false
+        settings.smartFormatting = true
```

Comment:
The `smartFormatting` parameter is certainly necessary for the test. But I suspect it would pass if we don't set the other parameters (which uses the defaults from the service instead). Can we try removing lines 307-316? I think that will help to show the "meat" of what this test is doing.